src/usrp_drivers/usrp_driver.cpp

```diff
@@ -36,7 +36,7 @@
 
 //Delay needed for before any set_time_commands will work.
 #define SET_TIME_COMMAND_DELAY 5e-3 // seconds
-#define TUNING_DELAY 300e-3 // seconds
+#define TUNING_DELAY 1e-3 // seconds
```

Comment:
How was this value chosen?

Reply:
Ah good question. Trial and error
[N200_Tuning_Timing_v2.ods](https://github.com/SuperDARNCanada/borealis/files/14762769/N200_Tuning_Timing_v2.ods)